neurodamus/node.py

```diff
@@ -1949,6 +1954,26 @@ def _instantiate_simulation(self):
         if MPI.rank == 0:
             self._merge_filesdat(n_cycles)
 
+    # -
+    def _instantiate_simulation(self):
+        """Initialize the simulation
+
+        - load targets
+        - check connections
+        - multicycle build model
+        """
+        # Keep the initial RSS for the SHM file transfer calculations
+        self._initial_rss = SHMUtil.get_node_rss()
+        print_mem_usage()
+
+        self.load_targets()
+
+        # Check connection block configuration and raise warnings for overriding
+        # parameters
+        SimConfig.check_connections_configure(self._target_manager)
+
+        self._multicycle_build_model()
```

Comment:
Is there a better name than `multicycle_build_model` ? Because the normal flow, the single cycle is also covered in this process. It sounds like neurodamus always does multicycle modeling.

Reply:
what about:
- `multicycle_build_model` -> `build_model`
- `build_model` -> `build_single_model`

Reply:
That sounds better. Please also remember to adjust the relevant docstrings.

Reply:
done